.circleci/config.yml

```diff
@@ -4,14 +4,14 @@ meta:
   xcode-versions: &xcode-versions
       matrix:
         parameters:
-          xcode-version: ["16.3", "16.4", "26.0"]
+          xcode-version: ["16.4", "26.0", "26.1.0"]
```

Comment:
The `major.minor` shortcut only appears to work when there is more than one image that would match. Not so much when `26.1.0` is the only game in town at the moment. 🤷